RobotsAndPencils/Astro
# Xcode 8 + Swift 2.3 Conversion

This upgrades the Astro project to use Xcode 8 and Swift 2.3 and packages it up to be released as version 2.0.0 so previous teams who haven't migrated yet can continue to use the 1.x version. It is also viewed as the first step towards migrating to Swift 3.

## How to test
1. Grab the latest code from Github and open up the project in Xcode 8
2. Review the minor code changes in our project to see what's been changed
3. Specially, check out the Route.swift change as the simple adjustment is to crash at runtime... to make it optional across the board is going to get messy but if you think it makes sense then lets start the discussion
3. Verify that there are no warnings/errors in the project
4. Run the tests... and watch them fail

NOTE: Unit tests will fail for the KeychainAccessSpec due to the an issue around Keychain entitlements (https://forums.developer.apple.com/thread/51071). Radar's have been filed by the looks of it so now its just a waiting game until those tests will clear.

## Areas of the app affected
- Dependent pods versions
- Project structure and Swift code conversion
